DataProcessing/datax-host/src/main/scala/datax/sink/BlobSinker.scala

```diff
@@ -126,7 +130,8 @@ object BlobSinker extends SinkOperatorFactory {
                      outputFolders: Map[String, String],
                      partitionId: Int,
                      compression: Boolean,
-                     loggerSuffix: String): Map[String, Int] = {
+                     loggerSuffix: String,
+                     blobStorageKey: broadcast.Broadcast[String]): Map[String, Int] = {
```

Comment:
Lets avoid passing broadcast variables as function params. Ideally, we should move the broadcast read code to hadoopClient

Reply:
Moved the broadcast read code to hadoopClient . As synced offline, we have to pass the broadcast variable as function param.